mob

```diff
@@ -200,6 +200,8 @@ docker_run = ["docker",
               # Give docker its own subdirectory under target so it doesn't trash
               # the host target directory, but will still be cleaned up with it.
               "--env", "CARGO_TARGET_DIR=" + mount_point + "/target/docker",
+              # This is required to run the fog unit tests
+              "--env", "TEST_DATABSE_URL=postgres://localhost",
```

Comment:
Spelling (I was wondering why it was still giving `panicked at 'env.TEST_DATABASE_URL must be set: NotPresent'`)
```suggestion
              "--env", "TEST_DATABASE_URL=postgres://localhost",
```